modules/oe_theme_content_project/src/Plugin/ExtraField/Display/OverallBudgetExtraField.php

```diff
@@ -74,7 +74,7 @@ public function getLabel() {
    * {@inheritdoc}
    */
   public function viewElements(ContentEntityInterface $entity) {
-    if ($entity->get('oe_project_budget')->isEmpty() && $entity->get('oe_project_eu_budget')->isEmpty()) {
+    if ($entity->get('oe_project_eu_budget')->isEmpty()) {
```

Comment:
We should not return empty arrays but use the isEmpty property.

Reply:
Ok this is good now but we still need to `return;` when there is nothing else to do.